app/Server/src/SERVERAPI/Startup.cs

```diff
@@ -46,6 +46,7 @@ public Startup(IWebHostEnvironment env)
 
             if (env.IsDevelopment())
             {
+                builder.AddJsonFile("secrets.json", optional: true, reloadOnChange: true);
```

Comment:
This is the only real change in this file and it is unrelated to migrations. Adding this line worked as a fix for a hard-to-address local development issue (only one dev had the issue) but it does not seem to have any side effects for working builds.

All other changes in this file are linting

Reply:
Yes I think this is OK as we want to unblock you. I would create a ticket and maybe leave a TODO in the code to pull this out when you offboard from NMP.